api/python/cellxgene_census/src/cellxgene_census/_release_directory.py

```diff
@@ -139,7 +139,7 @@ def get_census_version_description(census_version: str) -> CensusVersionDescript
 
 # TODO: Update example output and add example for use of release_flags
 def get_census_version_directory(
-    lts: Optional[bool] = None, retracted: Optional[bool] = None
+    lts: Optional[bool] = None, retracted: Optional[bool] = False
```

Comment:
not sure the docstring matches the default for `retracted`.

Also, do you really want the flags to accept `None`? Seems better to just force as keyword args, and explicitly default them, i.e.
>   def get_census_version_directory(*, lts: bool = False, retracted: bool = True)

Reply:
It's tricky because (I think) we want to support filtering releases in 3 ways:  flag is set (`flag=True`), flag is explicitly not set (`flag=False`), flag is ignored and can have either way (`flag=None`).

Reply:
ah, I did not understand the "do not filter" condition - I took "false" to mean "don't filter", which is clearly not what we want!

Given case,I agree with your ternary approach, but I would add the "no positionals" marker.  I also that if `lts` really defaults to true (rather than "ignore"), that the signature reflect this.

Reply:
added keyword-only marker to method signature and added full docs & examples. PTAL

> Also, if lts really defaults to true (rather than "ignore"), the signature should reflect this.

`lts` defaults to `None`/ignore, so I think we're good as is?